CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Added feature to use custom python files as modules to be used in the electron function
+- Added test coverage for executor resource deployment.
```

Comment:
This should come under the "Tests" header instead.

Reply:
Added Tests header under unreleased section